app/api/schema/search/facets/route.tsx

```diff
@@ -0,0 +1,8 @@
+import { NextResponse } from "next/server";
+import searchFacetsSchema from "../../../../src/data/schemas/searchFacetsSchema";
+
+export const GET = async () => {
```

Comment:
this is the intended api to use for the modal. 

idk if `facets` is the right term to use here? `filters` is kind of more implicit of what is actually being searched. 

regardless, interacting with the api to get the list of the filters in the modal would require a keyword search ie. `q=` and a way to identify the _category_ we are searching, ie. `category=`.

Reply:
If `facets` and `filters` are interchangeable, then the exact name doesn't matter too much, as long as we all know what it _should_ be called when talking about the options in the modal.